clients/java/api/openapi.yaml

```diff
@@ -2202,6 +2202,15 @@ paths:
         schema:
           type: string
         style: simple
+      - description: Ths source metarange to commit. Branch must not have unstaged
```

Comment:
What is the purpose of having this parameter in a GET operation? What do we expect to get here?

Reply:
Closing this comment and recreating it in `swagger.yaml` file. This file is generated by Java openapi generator..